SupportYourLocals.WPF/MainWindow.xaml.cs

```diff
@@ -577,14 +623,38 @@ private async Task LoadMarkerInformationWindow(string id)
             view.GroupDescriptions.Add(groupDescription);
         }
 
+        private void LoadMarketplaceInformationWindow(string id)
+        {
+            var items = new List<MarkerInformation>();
+
+            var locationData = sellerData.GetData(id);
+            foreach (var products in locationData.Result.Products)
+            {
+                foreach (var product in products.Value)
+                {
+                    items.Add(new MarkerInformation { ProductType = products.Key.ToString(), ProductCount = products.Value.Count, Product = product });
+                }
+            }
+            ListViewMarketplaceInformation.ItemsSource = items;
+
+            CollectionView view = (CollectionView)CollectionViewSource.GetDefaultView(ListViewMarketplaceInformation.ItemsSource);
+            PropertyGroupDescription groupDescription = new PropertyGroupDescription("ProductType");
+            view.GroupDescriptions.Add(groupDescription);
+        }
+
         private void CollapseMarkerInformation_Click(object sender, RoutedEventArgs e)
```

Comment:
This function is not necessary when there's one that does the exact same thing right above it, and it uses async correctly (asynchronous function results should be unpacked with "await", not by accessing the task's .Result value)